thresher/models.py

```diff
@@ -135,18 +155,21 @@ class Answer(models.Model):
     answer_id = models.IntegerField()
 
     # The question to which this answer belongs
-    question = models.ForeignKey(Question, related_name="answers")
+    question_id = models.OneToOneField(QuestionContent)
```

Comment:
I think this should still be a ForeignKey, a one to one filed is meant for one to one relationships. In this case however, each question can have multiple answers (since it's a multiple choice question) so it's a many to one relationship.